### DISCUSSION TITLE
Save path file on any model using Curator

### DISCUSSION THREAD

**Discussion by alibbali** [awcodes/filament-curator#447] [Q&A]
Hello all,

I'm curious about whether it's possible to store an uploaded file path directly in a Laravel model attribute. For instance, could I store a file path within a model like 'Speaker' under an attribute named 'picture'? I'd prefer not to utilize a separate media table for this purpose.

Additionally, I'm interested in whether Curator can be used to display all files located in the public/storage/ directory.

Thank you!

**Comment by awcodes** [comment]
Storing the path on the model would break the functionality of the plugin.

Currently, it doesn't support all files in the storage directory. It is planned for v4 though.
